test/TestApps/AppLifecycleTestApp/AppLifecycleTestApp.vcxproj

```diff
@@ -329,6 +329,10 @@
       <HintPath>$(OutDir)\..\ProjectReunion_DLL\Microsoft.ApplicationModel.Activation.winmd</HintPath>
       <IsWinMDFile>true</IsWinMDFile>
     </Reference>
+    <Reference Include="Microsoft.ProjectReunion">
+      <HintPath>..\..\..\BuildOutput\Debug\x64\ProjectReunion_DLL\Microsoft.ProjectReunion.winmd</HintPath>
+      <IsWinMDFile>true</IsWinMDFile>
+    </Reference>
```

Comment:
Why are we making changes to AppLifecycle test app?

Reply:
I think I was testing adding PR.winmd to the AppLifecycle test app. Will remove.